README.md

```diff
@@ -41,9 +41,10 @@ RAG is a method that tries to avoid hallucination by using information that is p
 4. The query generated from the last step is fed to an LLM model, and the **LLM model gives the user an answer based on the query** generated from the last step. The user receives this answer.
 
 <p align="center">
-<img width="1225" alt="Screenshot 2024-10-12 at 02 27 44" src="https://github.com/user-attachments/assets/bf6537c6-d50e-45cc-bb2f-caac0f3d90fe">
+	<img width="800" alt="Screenshot 2024-12-02 at 19 48 11" src="https://github.com/user-attachments/assets/9c502a2c-234d-448e-aba9-8f961c6e5c1a">
```

Comment:
I think the description of the generation part is outdated. That one should be responsible for generating answers instead of prompts

Reply:
That's right! I will change it